lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,186 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
```

Comment:
No, I need to be able to differentiate between the proxy being $null (which means, don't use a proxy even when the system was configured for one), and not having a proxy specified (which means, use the default system).

That's the whole point of `use_proxy`.